app/controllers/api/v1/rent_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Api
+  module V1
+    class RentController < ApiController
+      include Wor::Paginate
+
+      before_action :authenticate_user!
+
+      def index
+        render_paginated Rent, each_serializer: RentSerializer
+      end
+
+      def create
+        @rent = Rent.new(current_user, rent_params)
```

Comment:
```suggestion
        @rent = Rent.new(rent_params.merge(user_id: current_user.id))
```
If your version did not work.
Also check this guides for potencial exploits: https://rails-sqli.org/

Reply:
Ready

Reply:
Ready